portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
+  const aboutContainer = document.getElementById('aboutSpace');
+  aboutArr = aboutContainer.getElementsByTagName('p');
+
+  let aboutText; 
+
+  if (aboutArr.length > 0) {
+    aboutText = aboutContainer.getElementsByTagName('p')[0];
+  }
+
+  if (aboutText) {
+    // Change the text in the aboutSpace element 
+    if (aboutText.className === "about-hide") {
+      aboutText.className = "about-show";
+      aboutContainer.className = "background";
+    }
+    
+    else {
+      aboutText.className = "about-hide";
+      aboutContainer.className = "";
+    }
+  }
+}
+
+/* Shows pictures when the "Pictures" button is pressed. */
+function showPics() {
+  const imgs = ["ucla1.JPG", "ucla2.JPG", "book.JPG", "harrypotter.JPG"]
+
+  // Calculate a random index for the img array 
+  const randInt = (Math.floor(Math.random() * imgs.length));
+  const randImg = imgs[randInt];
+    
+  const div = document.getElementById('picSpace');
+  const imgArr = div.getElementsByTagName('img');
+
+  let picContainer;
+
+  if (imgArr.length > 0) {
+    picContainer = imgArr[0];
+  }
+
+  if (picContainer) {
+    // If pictures pressed, either remove / show picture depending on what is already there
```

Comment:
same here - no need to check picContainer if this if statement just checks imgArr.length